organic-green-theme.el

```diff
@@ -25,361 +25,376 @@
+   `(git-gutter:deleted ((t (:foreground ,organic-red-5))) t)
+   `(git-gutter:modified ((t (:foreground ,organic-blue-1))) t)
+   '(git-gutter-fr:added ((t (:foreground "PaleGreen3" :background "PaleGreen3"))) t)
+   '(git-gutter-fr:deleted ((t (:foreground "tomato1" :background "tomato1"))) t)
+   '(git-gutter-fr:modified ((t (:foreground "LightSkyBlue3" :background "LightSkyBlue3"))) t)
+
+   ;;; Org-Mode
+   `(org-table ((t (:foreground ,organic-teal))) t)
+   '(org-level-1 ((t (:inherit font-lock-function-name-face :bold t))) t)
+   '(org-level-2 ((t (:inherit font-lock-variable-name-face :bold t))) t)
+   '(org-level-3 ((t (:inherit font-lock-keyword-face :bold t))) t)
+   `(org-level-4 ((t (:foreground ,organic-green-8 :bold t))) t)
+   `(org-level-5 ((t (:foreground  ,organic-blue-4 :bold t))) t)
+   `(org-level-6 ((t (:foreground ,organic-teal :bold t))) t)
+   `(org-block ((,class (:foreground ,organic-fg))))
+   `(org-block-begin-line ((t (:foreground ,organic-blue-4))) t)
+   `(org-block-end-line ((t (:foreground ,organic-blue-4))) t)
+
+   ;;; Misc
+   `(nxml-element-local-name ((t (:foreground ,organic-blue-3 :weight normal))) t)
+   `(yas-field-highlight-face ((t (:background ,organic-yellow-2))))
+   `(idle-highlight ((t (:background ,organic-yellow-0))) t)
+   `(comint-highlight-prompt ((t (:foreground ,organic-blue-4 :weight bold))) t)
+   `(flx-highlight-face  ((t (:foreground ,organic-blue-3 :bold t :underline t))) t)
+
+   ;; powerline <TODO>
+   `(powerline-active1 ((t (:background ,organic-alum-3 :inherit mode-line))) t)
+   `(powerline-active2 ((t (:background ,organic-alum-2 :inherit mode-line))) t)
+   '(powerline-inactive1  ((t (:background "grey70" :inherit mode-line-inactive))) t)
+   '(powerline-inactive2  ((t (:background "grey80" :inherit mode-line-inactive))) t)
+
+   ;; tabbar <TODO>
+   '(tabbar-button ((t :inherit tabbar-default :box (:line-width 1 :color "gray72"))))
+   `(tabbar-modified ((t (:inherit tabbar-default :foreground "#118811"
+                                   :bold t
+                                   :box (:line-width 1 :color ,organic-white
+                                                     :style released-button)))))
+   `(tabbar-selected ((t :inherit tabbar-default
+                         :box (:line-width 1 :color ,organic-white :style pressed-button)
+                         :foreground ,organic-alum-6 :bold t)))
+   '(tabbar-selected-modified ((t :inherit tabbar-selected)))
+
+   ;;; Web-Mode
+   `(web-mode-current-element-highlight-face ((,class (:background ,organic-green-1))))
+   `(web-mode-html-tag-face ((t (:foreground ,organic-gray-6))) t)
+   `(web-mode-html-attr-name-face ((t (:foreground ,organic-blue-5))) t)
+   `(web-mode-doctype-face ((t (:foreground ,organic-blue-4))) t)
+   `(web-mode-comment-face ((t (:foreground ,organic-gray-4)) t))
+   `(web-mode-css-selector-face ((t (:foreground ,organic-teal))) t)
+   `(web-mode-function-call-face ((t :inherit ,organic-fg)))
```

Comment:
The face can't be inherited from a color.
You should use either `:foreground ,organic-teal` or `:inherit default`.

Reply:
I explicitly set the `foreground` value.